blockchains/utxo/utxo_worker.js

```diff
@@ -45,14 +47,35 @@ class UtxoWorker extends BaseWorker {
     });
   }
 
+  async sendRequestWithRetry(method, params) {
+    let retries = 0;
+    let retryIntervalMs = 0;
+    while (retries < MAX_RETRIES) {
+      try {
+        const response = await this.sendRequest(method, params).catch(err => Promise.reject(err));
+        if (!response.error && response.result !== null) {
+          return response;
+        }
+      } catch(err) {
+        retries++;
+        retryIntervalMs += 1000;
+        logger.error(
+          `Try block in sendRequest for ${method} failed. Reason: ${err.toString()}. Waiting ${retryIntervalMs} and retrying for ${retries} time`
+          );
+        await new Promise((resolve) => setTimeout(resolve, retryIntervalMs));
```

Comment:
But this way you are missing the case where there is no exception but the response is an error. In your previous version you triggered a retry in this case. My suggestion was to move the common code after the catch() but still in the loop.